flaskr/app.py

```diff
@@ -29,7 +29,31 @@ def root():
 
     return app
 
-
+# Instantiate Flask app and SQLAlchemy ORM
 app = create_app()
 db = SQLAlchemy(app)
-from .model import Language
+
+# Define models in relation to instantiated ORM
+class Language(db.Model):
+    __tablename__ = 'languages'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String())
+    methods = db.relationship('Method', backref='language')
+
+    def __repr__(self):
+        return '<Language %r>' % self.name
```

Comment:
Looks like models now live directly in the .app?

Reply:
Yep; see my below comment for one reason, the other reason was that this resolved all the issues with importing the model file (at least until we learn how to do all of that better).